docs/client-development/mandate.rst

```diff
@@ -0,0 +1,526 @@
+             &rol__machtiging=eigen
+
+
+    Additionally ``rol.omschrijvingGeneriek`` can be used to determine the "initiator"
+    of the case
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__kvk_Nummer=<own_kvk>
+             &rol__machtiging=eigen
+             &rol__omschrijvingGeneriek=initiator
+
+
+2. Show me my cases (based on my KVK nummer) opened by an authorizee on my behalf:
+
+    .. code::
+
+        # organization
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__kvk_Nummer=<own_kvk>
+             &rol__machtiging=machtiginggever
+
+    .. code::
+
+        # branch
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__vestiging__kvkNummer=<own_kvk>
+             &rol__machtiging=machtiginggever
+
+
+    Additionally ``rol.omschrijvingGeneriek`` can be used to determine the "belanghebbende"
+    of the case.
+
+    .. code::
+
+        # organization
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__kvk_Nummer=<own_kvk>
+             &rol__omschrijvingGeneriek=belanghebbende
+
+
+3. Show me the cases (based on my KVK nummer) where I represent another party:
+
+    .. code::
+
+        # organization
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__kvk_Nummer=<own_kvk>
+             &rol__machtiging=gemachtigde
+
+    .. code::
+
+        # branch
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__vestiging__kvkNummer=<own_kvk>
+             &rol__machtiging=gemachtigde
+
+
+    It's also possible to filter cases based on the level of assurance and to exclude
+    results where levelOfAssurance is below some required value.
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__kvk_Nummer=<own_kvk>
+             &rol__machtiging=gemachtigde
+             &rol__machtiging__loa=urn:etoegang:core:assurance-class:loa2plus
+
+
+**Filters for rollen**
+
+It's also possible to make requests with such filters for ``zaken/api/v1/rollen`` endpoint
+to retrieve details of the parties. For example, show me the rollen (based on my BSN) where I represent another party:
+
+    .. code::
+
+        GET /zaken/api/v1/rollen?
+             betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &machtiging=gemachtigde
+             &machtiging__loa=urn:oasis:names:tc:SAML:2.0:ac:classes:MobileTwoFactorContract
+
+
+.. note::
+
+   If it's possible that your case has several autorizees and representees, use ``zaken/api/v1/rollen`` endpoint
+   for searching such cases. Filters for the ``zaken/api/v1/zaken`` endpoint don't depend on each other, so
+   it's possible to show cases where one query parameter belongs to one role and another parameter belongs to
```

Comment:
Isn't that a bit odd? I'd expect all my filters to be AND-ed together and applied to each role as a consumer of the API.

Reply:
Let's discuss it

For now we have a lot of `rol__*` query params for `zaken/api/v1/zaken` endpoint:
* rol__betrokkeneType
* rol__betrokkene
* rol__omschrijvingGeneriek
* rol__betrokkeneIdentificatie__* query params for bsn, rsin etc,

All of them are not ANDed, so if you add query params 
`?rol__betrokkeneType=vestiging&rol__omschrijvingGeneriek=initiator` you receive zaken which have one role "vestiging" and one rol "initiator" and it's not required to be the same rol. 
This is how it works currently in Open Zaak. 

In this PR we add new query params:
* rol__machtiging
* rol__machtiging__loa

If query params for rols should be ANDed together there is a question which params should it be? All `rol__*` query params or only `rol__machtiging*` query params or `rol__machtiging* + rol__betrokkeneType` ? 
I don't know what behavior is expected here, this is why I've decided to:
* keep the behavior of query params as it was before
* add similar query params in the Rol resource
* highlight it in the documentation

I can make some query params ANDed together, but I don't know which ones

Reply:
I suppose this path of django docs describes this behaviour for filtering on reverse fk relations https://docs.djangoproject.com/en/4.2/topics/db/queries/#spanning-multi-valued-relationships

Reply:
I'd like to discuss htis in more depth in a call or short meeting int he office!

Reply:
Discussed: let's make all query params ANDed together

Reply:
Implemented in `FilterSetWithGroups`

Reply:
changed the filterset and added related tests for it (`FilterRolSpanningTests`)